src/genophenocorr/preprocessing/_config.py

```diff
@@ -267,30 +269,33 @@ def load_phenopacket_folder(
 
 def _summarize_validation(policy: str,
                           notepad: NotepadTree,
-                          indent: int = 2):
-    lines = [f'Validated under {policy} policy']
+                          indent: int = 2) -> typing.Tuple[typing.Sequence[str], typing.Sequence[str]]:
+    warn_lines = [f'Validated under {policy} policy']
+    error_lines = []
     n_errors = sum(node.error_count() for node in notepad.iterate_nodes())
     n_warnings = sum(node.warning_count() for node in notepad.iterate_nodes())
     if n_errors > 0 or n_warnings > 0:
-        lines.append('Showing errors and warnings')
+        warn_lines.append('Showing errors and warnings')
         for node in notepad.iterate_nodes():
             if node.has_errors_or_warnings(include_subsections=True):
                 # We must report the node label even if there are no issues with the node.
                 l_pad = ' ' * (node.level * indent)
-                lines.append(l_pad + node.label)
+                warn_lines.append(l_pad + node.label)
                 if node.has_errors():
-                    lines.append(l_pad + ' errors:')
+                    error_lines.append(l_pad + ' errors:')
                     for error in node.errors():
-                        lines.append(l_pad + ' ' + error.message
+                        error_lines.append(l_pad + ' ' + error.message
                                      + (f'. {error.solution}' if error.solution else ''))
                 if node.has_warnings():
-                    lines.append(l_pad + ' warnings:')
+                    warn_lines.append(l_pad + ' warnings:')
                     for warning in node.warnings():
-                        lines.append(l_pad + ' ·' + warning.message
+                        warn_lines.append(l_pad + ' ·' + warning.message
                                      + (f'. {warning.solution}' if warning.solution else ''))
     else:
-        lines.append('No errors or warnings were found')
-    return lines
+        warn_lines.append('No errors or warnings were found')
```

Comment:
Here we found no errors. We can print the corresponding text to STDOUT (no red).